packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts

```diff
@@ -53,7 +51,7 @@ const _useOrganizedParticipants = (props: OrganizedParticipantsArgs): OrganizedP
           participants: videoParticipants,
           dominantSpeakers,
           lastVisibleParticipants: visibleGridParticipants.current,
-          maxDominantSpeakers: maxRemoteVideoStreams
+          maxDominantSpeakers: maxRemoteVideoStreams || 4
```

Comment:
can we remove the `|| 4` and just ensure that this is a required prop to this hook and the value is always passed in? There's a lot of defaults floating around that aren't tied to constants, if `4` ever needs updated it difficult to find all the spots where to update.

Reply:
I believe there was a decision made that we were to use the old constant. so if we are providing a default should OVC fail the video gallery should fall back to the original value of `DEFAULT_MAX_REMOTE_VIDEOSTREAMS`